### ISSUE TITLE
Installation of Selenium IDE Fails

### ISSUE THREAD

**Issue by GoogleCodeExporter** [prajaktamoghe/selenium-vba#85]
```
Operating system and version (32/64bit) : Windows 7 64 bit

The installation of the IDE fails because it is expecting firefox.exe to be 
installed on the c: drive. However, on my system it is installed on the e: 
drive.

I guess the software should check where firefox is installed before attempting 
to install the IDE?

Great software by the way... I love it! Thank you!
```

Original issue reported on code.google.com by `gshym...@gmail.com` on 17 Aug 2014 at 10:43

**Comment by GoogleCodeExporter** [comment]
```
By the way, as a temporary workaround I used the utility "junction" (downloaded 
from the link below) to create a symbolic link from c:\...\Mozilla Firefox to 
e:\...\Mozilla Firefox. Once I created the link the install worked ok. Note 
that the utility "junction" MUST be run from a command line that was started in 
administrator mode.

http://technet.microsoft.com/en-us/sysinternals/bb896768.aspx 
```

Original comment by `gshym...@gmail.com` on 17 Aug 2014 at 10:58

**Comment by GoogleCodeExporter** [comment]
```
Actually, it's not restricted to the C drive, but to the default installation 
path:
%ProgramFiles%\Mozilla Firefox\firefox.exe
You can also install the extension manually by dropping the file 
"selenium-ide.xpi" in Firefox.
A fix would be to look up the path from the registry.
```

Original comment by `florentbr` on 21 Aug 2014 at 7:05
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in v1.0.20.0 (Now takes the path from the registry)
```

Original comment by `florentbr` on 18 Sep 2014 at 4:32
- Changed state: **Fixed**
